client/state/reader/lists/test/selectors.js

```diff
@@ -194,6 +195,67 @@ describe( 'selectors', () => {
 		} );
 	} );
 
+	describe( '#getMatchingItem()', () => {
+		const feed = {
+			feed_ID: 1,
+		};
+		const feedItem = {
+			feed_URL: 'https://www.example.com/rss',
+			feed_ID: 1,
+		};
+		const site = {
+			meta: { data: { site: { blog_ID: 0 } } },
+			site_ID: 1,
+		};
+		const tag = {
+			meta: { data: { tag: { blog_ID: 0 } } },
+			tag_ID: 1,
+		};
+		// state.reader.feeds.items
+		const state = {
+			reader: {
+				feeds: { items: { 1: feedItem } },
+				lists: { listItems: { 1: [ feed, site, tag ] } },
+			},
+		};
+		test( 'should return false if the list does not exist', () => {
+			expect( getMatchingItem( state, { feedUrl: 'www.example.com', listId: 2 } ) ).to.eql( false );
```

Comment:
Would be good to use Jest assertions for these new tests. 

If you want to leave the old tests using Chai assertions for now, elsewhere we have done:

`import { expect as chaiExpect } from 'chai';`

... and switched the older tests to use chaiExpect instead.